src/main.jsx

```diff
@@ -6,12 +6,19 @@ import {
   RouterProvider,
 } from "react-router-dom";
 import TrendingMedia from "./views/trendingMedia";
+import Root from "./views/root";
 
 
 const router = createBrowserRouter([
   {
     path: "/",
-    element: <TrendingMedia />
+    element: <Root />,
+    children: [
+      {
+        path: "/",
```

Comment:
Here you can just set it to index: true and it will have the same path as the parent route:
```suggestion
        index: true,
```